app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -590,6 +591,15 @@ public function newAction($entity = null)
         $sections    = $model->getBuilderComponents($entity, 'sections');
         $sectionForm = $this->get('form.factory')->create('builder_section');
 
+        $pluginsAssets = [];
+        $plugins       = $this->get('mautic.helper.plugin.builder')->getBuilderPlugins();
+
+        if ($plugins) {
+            foreach ($plugins as $name => $config) {
+                $pluginsAssets[] = '<script src="/plugins/'.$name.'/Assets/js/index.js"></script>';
```

Comment:
I don't think this will work for installations in a subdirectory. Please use the path helper to build the URLs.

Also ideally the plugin config should have all assets listed or loaded automatically from the folder if some JS files exists there.

Reply:
Hello, I'll be taking over this work from here. If I understand correctly in your opinion this code can be completely stripped out with no other changes then?